src/models/Playlist.ts

```diff
@@ -0,0 +1,11 @@
+export type PlaylistQuery = {
```

Comment:
Can we separate both types into different files?

Reply:
yes, actually I consider It'd be better to rename the folder to "types"